### ISSUE TITLE
Softmax across all keypoints?

### ISSUE THREAD

**Issue by Yishun99** [salihkaragoz/pose-residual-network-pytorch#19]
```python
class Flatten(nn.Module):
    def forward(self, input):
        return input.view(input.size(0), -1)

class PRN(nn.Module):
    def __init__(self,node_count,coeff):
        ...
        self.softmax   = nn.Softmax(dim=1)

    def forward(self, x):
        res = self.flatten(x)
        ...
        out = self.add(out,res)  # [N,H*W*C]
        out = self.softmax(out)
        out = out.view(out.size()[0],self.height, self.width, 17)

        return out

```

**Comment by salihkaragoz** [comment]
Hi,
We tried both softmax per keypoint and across keypoints. Computing the softmax activations per keypoint channel is the intuitive way. However, during the experiments, we get similar results with both and use the simplest one.
I am closing this issue, feel free for further questions,
Best
